core/src/main/java/com/google/zxing/multi/GenericMultipleBarcodeReader.java

```diff
@@ -86,8 +86,8 @@ private void doDecodeMultiple(BinaryBitmap image,
       return;
     }
     boolean alreadyFound = false;
-    for (Result existingResult : results) {
-      if (existingResult.getText().equals(result.getText())) {
+    for (Result result : results) {
```

Comment:
Wait, this won't even compile. There's already a `result` in scope which is why this one is named differently.